Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/MarkdownRenderedEventArgs.cs

```diff
@@ -18,6 +18,7 @@ namespace Microsoft.Toolkit.Uwp.UI.Controls
     /// Arguments for the OnMarkdownRendered event which indicates when the markdown has been
     /// rendered.
     /// </summary>
+    [Obsolete("Use Microsoft.Toolkit.Services.Markdown.MarkdownRenderedEventArgs instead.")]
     public class MarkdownRenderedEventArgs : EventArgs
```

Comment:
This class should just extend the new class. Wouldn't the new class require developers to include the new package as well in order to use the event?